app/models/notification.rb

```diff
@@ -1,9 +1,15 @@
 class Notification < ApplicationRecord
   belongs_to :user
 
+  @@NOTIFICATION_TYPES = {LIKES: "likes",COMMENTS:"comments",FOLLOW:"follow",UNFOLLOW:"unfollow",ACCEPTANCE:"follow_acceptance"}
+
+  # getter
+  def self.NOTIFICATION_TYPES
+    @@NOTIFICATION_TYPES
+  end
   
   def self.create_notification(parameters)
-    @notification=Notification.new(user_id:parameters[:user_id],actor_id:parameters[:actor_id],notification_type:parameters[:notification_type],target_type:parameters[:target_type],target_id:parameters[:target_id])
-    @notification.save
+    notification=Notification.new(user_id:parameters[:user_id],actor_id:parameters[:actor_id],notification_type:parameters[:notification_type],target_type:parameters[:target_type],target_id:parameters[:target_id])
+    notification.save
```

Comment:
Please indent you code a bit.  On one line don't cover more than 80 columns , in that case you have scroll to right to see complete code. And same problem occurs in github. Please do this.

Reply:
done